config/katello.migrations/160608111430-additional-foreman-proxy.rb

```diff
@@ -1,4 +1,5 @@
 def move(name, new_name=nil, default=nil)
+  return if answers['capsule'].key?(name)
```

Comment:
Would we need similar statement in 160120212701-foreman-proxy.rb & 160601202256-additional-foreman-proxy.rb ?